installer/install.sh

```diff
@@ -17,6 +17,40 @@ oc_version_comparison=${VER_LT}
 # 0 - =
 # 1 - >
 # 2 - <
+
+function spinner() {
```

Comment:
is a spinner really necessary?

Reply:
Hi @maleck13,

To run `oc cluster up` can take a while then in order to improve the experience yes. It is too bad for the user don't know why it is stoped in some process without the information that it is processing.